ethcore/res/ethereum/foundation.json

```diff
@@ -163,7 +163,13 @@
 		"eip658Transition": "0x42ae50",
 		"eip145Transition": "0x6f1580",
 		"eip1014Transition": "0x6f1580",
-		"eip1052Transition": "0x6f1580"
+		"eip1052Transition": "0x6f1580",
+		"eip1283Transition": "0x0",
+		"eip1283DisableTransition": "0x0",
+		"eip1283ReenableTransition": "0x8a61c8",
```

Comment:
I think

```bash
	    "eip1283Transition": "0x0",
		"eip1283DisableTransition": "0x0",
		"eip1283ReenableTransition": "0x8a61c8",
```
is equivalent to 

```bash
	    "eip1283Transition": "0x8a61c8",
```

 I copy-pasted it from `goerli chain spec` and just changed the block number

Reply:
We can just use `eip1283Transition` and skip `eip1283DisableTransition` and `eip1283ReenableTransition`.